CONTRIBUTING.md

```diff
@@ -74,20 +74,20 @@ Formatting is not required during the development stage but it is encouraged sin
 
 ## Testing
 
-Each subproject will have its own testing requirements. Please read the README for each subproject to see what is required.
+Each subproject will have its testing requirements. Please read the README for each subproject to see what is required.
 
-There are also integration tests which will run on your PR to ensure that it passes all the tests.
+There are also integration tests that will run on your PR to ensure that it passes all the tests.
```

Comment:
Perhaps this sentence should be reworded to be like
`there are integration tests that will run on your pr to ensure nothing else broke during your implementation`